generate/param_generator.go

```diff
@@ -483,6 +483,10 @@ func (slP StringSliceParam) GetStringSlice() []string {
 	return viper.GetStringSlice(slP.name)
 }
 
+func (slP StringSliceParam) GetName() string {
```

Comment:
Can you create an issue for someone to go through and change all our viper calls to use this instead of the string names? It's busy work, but it would be nice to implement at some point.

Reply:
Issue created here: https://github.com/PelicanPlatform/pelican/issues/1526